src/node_worker.cc

```diff
@@ -66,11 +55,11 @@ Worker::Worker(Environment* env,
                std::shared_ptr<PerIsolateOptions> per_isolate_opts,
                std::vector<std::string>&& exec_argv)
     : AsyncWrap(env, wrap, AsyncWrap::PROVIDER_WORKER),
-      url_(url),
       per_isolate_opts_(per_isolate_opts),
       exec_argv_(exec_argv),
       platform_(env->isolate_data()->platform()),
-      profiler_idle_notifier_started_(env->profiler_idle_notifier_started()),
+      // XXX(joyeecheung): should this be per_process::v8_is_profiling instead?
+      start_profiler_idle_notifier_(env->profiler_idle_notifier_started()),
```

Comment:
I think the main question for the `XXX` is whether we want to keep the semi-private (?) `process._startProfilerIdleNotifier` / ` process._stopProfilerIdleNotifier` methods, right? I don’t quite know why these were added (f649626c6faef9d74c1e77608f952b06f4c5205b) … maybe @bnoordhuis remembers?

Reply:
I think that answers my question. See https://github.com/nodejs/node/issues/19009 (it's used in https://github.com/googleapis/cloud-profiler-nodejs/pull/248)